packages/image_picker/image_picker_android/android/build.gradle

```diff
@@ -42,7 +42,10 @@ android {
         implementation 'androidx.core:core:1.9.0'
         implementation 'androidx.annotation:annotation:1.3.0'
         implementation 'androidx.exifinterface:exifinterface:1.3.6'
-        implementation 'androidx.activity:activity:1.6.1'
+        implementation 'androidx.activity:activity:1.7.0'
+        // org.jetbrains.kotlin:kotlin-bom artifact purpose is to align kotlin stdlib and related code versions.
+        // See: https://youtrack.jetbrains.com/issue/KT-55297/kotlin-stdlib-should-declare-constraints-on-kotlin-stdlib-jdk8-and-kotlin-stdlib-jdk7
+        implementation(platform("org.jetbrains.kotlin:kotlin-bom:1.8.10"))
```

Comment:
if this is meant to be temporary, can you add a TODO here?

Reply:
This fix is in a couple of places at this point, so I'll add discussion about when this can be removed to the tracking issue
https://github.com/flutter/flutter/issues/125062